rest_framework/metadata.py

```diff
@@ -107,10 +114,9 @@ def get_serializer_info(self, serializer):
             # If this is a `ListSerializer` then we want to examine the
             # underlying child serializer instance instead.
             serializer = serializer.child
-        return OrderedDict([
-            (field_name, self.get_field_info(field))
-            for field_name, field in serializer.fields.items()
-        ])
+        return OrderedDict([(field_name, self.get_field_info(field))
+                            for field_name, field in
+                            serializer.fields.items()])
```

Comment:
That's the only indentation style that works within the 79 character limit.  Since I only noticed that we're ignoring the line length PEP8 rule after the fact, we don't need to worry about the indentation style changing.  

Still, that is the only way to make the linter work and not ignore the line length rule (which I would be in favor of doing).  I think the best course of action is to continue ignoring the line length, do everything else, and then decide about the line length at some future date.  I would be in favor of the 79 character limit though - it's really easier once you take the plunge...